alby.go

```diff
@@ -93,6 +93,21 @@ func (svc *AlbyOAuthService) MakeInvoice(ctx context.Context, senderPubkey strin
 		}).Errorf("App not found: %v", err)
 		return "", "", err
 	}
+
+	// amount provided in msat, but Alby API currently only supports sats. Will get truncated to a whole sat value
+	var amountSat int64 = amount / 1000
+	// make sure amount is not converted to 0
+	if (amount > 0 && amountSat == 0) {
+		svc.Logger.WithFields(logrus.Fields{
+			"senderPubkey":    senderPubkey,
+			"amount":          amount,
+			"description":     description,
+			"descriptionHash": descriptionHash,
+			"expiry":          expiry,
+		}).Errorf("Value must be 1 sat or greater");
+		return "", "", errors.New("Value must be 1 sat or greater")
```

Comment:
what is the name of the field that the user is passing in? "value"?

Reply:
Sorry, it's "amount". I will update this.